ui/src/main/webapp/src/app/executions/executions-list.component.html

```diff
@@ -30,7 +30,16 @@
         </thead>
         <tbody>
         <tr *ngFor="let execution of sortedExecutions" [class]="getClass(execution)" class="execution-row">
-            <td><a [routerLink]="['/projects', execution.projectId, execution.id, 'execution-details']">#{{execution.id}}</a></td>
+            <td>
+                <span *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports; else elseExecutionState">
+                    <a  class="pointer link" target="_blank" href="{{formatStaticReportUrl(execution)}}" title="Show reports">
+                        #{{execution.id}} <i class="fa fa-external-link fa-fw"></i>
```

Comment:
I think that the more accepted place for that icon is at the end of a link. I'll poke around and see if I can find anything to back that up though.